js/ui/camera.js

```diff
@@ -746,11 +750,12 @@ util.extend(Camera.prototype, /** @lends Map.prototype */{
         return bearing;
     },
 
-    _updateEasing: function(duration, zoom, bezier) {
```

Comment:
Yes, entirely dead code. I had to do `git bisect` combined with project search to find the commit that made it so.